java-it-tests/common/src/main/java/common/CassandraResult.java

```diff
@@ -43,6 +43,19 @@ public SuccessStatus() {
         }
     }
 
+    public static class ErrorMessageStatus extends ErrorStatus {
```

Comment:
Did this in favor of creating a `ErrorStatus` for each response time as they only return error messages from the exception, though some exceptions may surface specific things, so I may add more to this (need to look).